docs/manual/comparison_with_rdatatable.rst

```diff
@@ -0,0 +1,354 @@
+
+``data.table``::
+
+    library(data.table)
+
+    DT = data.table(x=rep(c("b","a","c"),each=3),
+                    y=c(1,3,6), v=1:9)
+
+
+``datatable``::
+
+    from datatable import dt, f, g, by, update, join, sort
+    import numpy as np
+
+    DT = dt.Frame({"x":np.repeat(["b","a","c"],3),
+                   "y": [1,3,6]*3,
+                   "v": range(1,10)}) # datatable
+
+
+
+
+===============================   ============================================ ==================================================
+data.table                           datatable                                   Action
+===============================   ============================================ ==================================================
+``DT[2]``                            ``DT[1, :]``                               Select 2nd row
+``DT[2:3]``                          ``DT[1:3, :]``                             Select 2nd and 3rd row
+``DT[3:2]``                         ``DT[[2,1], :]``                            Select 3rd and 2nd row
+``DT[.N]``                          ``DT[-1, :]``                               Select the last row
+``DT[y>2]``                         ``DT[f.y>2, :]``                            All rows where ``y > 2``
+``DT[y>2 & v>5]``                   ``DT[(f.y>2) & (f.v>5), :]``                Compound logical expressions
+``DT[!2:4]`` or ``DT[-(2:4)]``      ``DT[[0, slice(4, None)], :]``              All rows other than rows 2,3,4
+``DT[order(x), ]``                  | ``DT.sort("x")`` or                       Sort by column ``x``, ascending
+                                    | ``DT[:, :, sort("x")]``
+``DT[order(-x)]``                   | ``DT.sort(-f.x)`` or                      Sort by column ``x``, descending
+                                    | ``DT[:, :, sort(-f.x)]``
+``DT[order(x, -y)]``                | ``DT.sort(x, -f.y)`` or                   Sort by column ``x`` ascending, ``y`` descending
+                                    | ``DT[:, :, sort(x, -f.y)]``
+===============================   ============================================ ==================================================
+
+Note the use of the ``f`` symbol when performing computations or sorting in descending order. You can read more about :ref:`f-expressions`.
+
+Selecting Columns
+-----------------
+
+=========================================== ============================================ ==============================================
+data.table                                           datatable                                                  Action
+=========================================== ============================================ ==============================================
+``DT[, .(v)]``                               ``DT[:, 'v']``                              Select column ``v``
+``DT[, .(x,v)]``                             ``DT[:, ['x', 'v']]``                       Select multiple columns
```

Comment:
I think there should also be an example of selecting a column whose name is in a variable, for example if `var='v'` then it's `DT[var]` in Python, and `DT[, ..var]` in R.